src/Sulu/Bundle/SecurityBundle/Entity/AccessControl.php

```diff
@@ -38,7 +38,7 @@ class AccessControl implements AccessControlInterface
     /**
      * The id of the model this access control rule applies to.
      *
-     * @var int
+     * @var string
```

Comment:
I don't think we can change this to string I think for BC I would go here with `string|int` and go in the `getEntityId()` with a `is_numeric` check, because smebody could made here a typesafe check with `===` to the access control object so I would go here with:

```php
public function getEntityId()
{
     if (\is_numeric($this->entityId)) {
           return (int) $this->entityId;
     }
     
     return $this->entityId;
}
```